build_targets

```diff
@@ -4,7 +4,7 @@
 beryllium userdebug yes [recovery.img]
 #cheeseburger user yes no
 #clover userdebug yes no
-#courbet user yes yes
+courbet user yes [boot.img, recovery.img]
```

Comment:
you sure you need both?

Reply:
not fully important, it was to provide boot.img too. we can drop it